gitea/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       PROXY_AUTH_ADD: "false"
 
   server:
-    image: gitea/gitea:1.16.8-rootless@sha256:8a6ad22694c76122ae075af1153a544577724c45cb45af44963aa78dc179cc46
+    image: gitea/gitea:latest
```

Comment:
We can't accept the usage of `latest` as a Docker image tag because this would allow updates to simply get rolled out without at least some check from us.